src/Parser/DecimalParserTrait.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace Money\Parser;
+
+use Money\Currency;
+use Money\Exception\ParserException;
+use Money\Money;
+use Money\Number;
+
+trait DecimalParserTrait
+{
+    public static $decimalPattern = '/^(?P<sign>-)?(?P<digits>0|[1-9]\d*)?\.?(?P<fraction>\d+)?$/';
```

Comment:
This should be a const IMHO

Reply:
Traits can not have constants :)

Reply:
In that case, I would make it protected.

Reply:
Done.